src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1490,34 +1494,35 @@ class Simplex_tree {
    * performed on these values during the computation of extended persistence.
    * @param[in] dgm Persistence diagram obtained after calling `extend_filtration()`, 
    * `initialize_filtration()`, and `Gudhi::persistent_cohomology::Persistent_cohomology< FilteredComplex, CoefficientField >::compute_persistent_cohomology()`.
+   * @param[in] efd Structure containing the minimum and maximum values of the original filtration
```

Comment:
I think the main defining trait of this parameter is that it is "what extend_filtration returned".

Reply:
Agreed. Done.